volatility3/framework/plugins/windows/ldrmodules.py

```diff
@@ -82,45 +88,115 @@ def filter_function(x: interfaces.objects.ObjectInterface) -> bool:
                 init_mod = init_order_mod.get(base, None)
                 mem_mod = mem_order_mod.get(base, None)
 
-                yield (
-                    0,
-                    [
-                        int(proc.UniqueProcessId),
-                        str(
-                            proc.ImageFileName.cast(
-                                "string",
-                                max_length=proc.ImageFileName.vol.count,
-                                errors="replace",
-                            )
-                        ),
-                        format_hints.Hex(base),
-                        load_mod is not None,
-                        init_mod is not None,
-                        mem_mod is not None,
-                        mapped_files[base],
-                    ],
-                )
+                load = renderers.NotApplicableValue()
+                init = renderers.NotApplicableValue()
+                mem = renderers.NotApplicableValue()
+
+                try:
+                    if load_mod:
+                        load = "{0} : {1}".format(
+                            load_mod.FullDllName.get_string(),
```

Comment:
Do these two values differ, or is the `BaseDllName` part of the `FullDllName`?  Either way we try hard to avoid textually combining data into a single column, because it means that a program using volatility as a library has to do work to split the data back up into its components.  As such, I'd either change the default column entry to the full one (and forget the flag) or if both bits of information are important, then I'd output a second column.

It's slightly more effort, but it will make it much easier for other code to process the results, and humans will still be able to read it.  Since we have version numbers we can change the default output of a plugin without too much issue (and people can check the version number if they depend on specific columns).